.yarnrc.yml

```diff
@@ -15,14 +15,16 @@ npmAuditExcludePackages:
   # pending | low      | GHSA-j58c-ww9w-pwp5 | angular <=1.8.3              | 1.8.3 brought in by manageiq-ui-classic@workspace:.
   # pending | low      | GHSA-m9gf-397r-hwpg | angular >=1.3.0-rc.4 <=1.8.3 | 1.8.3 brought in by manageiq-ui-classic@workspace:.
   # pending | low      | GHSA-mqm9-c95h-x2p6 | angular <=1.8.3              | 1.8.3 brought in by manageiq-ui-classic@workspace:.
+  - angular-sanitize
+  # pending | moderate | GHSA-4p4w-6hg8-63wx | angular-sanitize >=1.3.1 <=1.8.3 | 1.8.3 brought in by manageiq-ui-classic@workspace:.
   - bootstrap
-  # pending | moderate | GHSA-9mvj-f7w8-pvh2 | bootstrap >=2.0.0 <=3.4.1 | 3.4.1 brought in by eonasdan-bootstrap-datetimepicker@virtual:6557af66f1a3f9d81b30dd080e0c619c011728fa0bded31959e34111c8a06d5f82c760d19a3100c59f98831cc7e73d48a91c41d41de6db59b14a282cb6169d79#npm:4.17.49
+  # pending | moderate | GHSA-9mvj-f7w8-pvh2 | bootstrap >=2.0.0 <=3.4.1 | 3.4.1 brought in by eonasdan-bootstrap-datetimepicker@virtual:5c616db0543848e1d5b60ddf9f1567af3b6c22b5b10c5930237ca94f8aa79437779f5e0c9acab451dae388870ac6914a83389ac4c8165cb0b02e27d6c3d96535#npm:4.17.49
```

Comment:
Was this change expected/needed?  I was expecting only `angular-sanitize` to be added.

Reply:
This is common if we have a moving target like a SHA as the script keeps the entire list up to date.